tests/python/pants_test/backend/jvm/tasks/test_ivy_updates_integration.py

```diff
@@ -0,0 +1,69 @@
+# coding=utf-8
+# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+import os
+from textwrap import dedent
+
+from pants.base.build_environment import get_buildroot
+from pants_test.pants_run_integration_test import PantsRunIntegrationTest
+from pants.util.contextutil import temporary_dir
+
+
+# TODO: re-enable these tests after the ivy-depdendency-update-checker has been released
+class IvyUpdatesIntegrationTest(PantsRunIntegrationTest):
+
+  def xxtest_with_no_depedencies(self):
+    with temporary_dir(root_dir=get_buildroot()) as tmpdir:
+      with open(os.path.join(tmpdir, 'BUILD'), 'w+') as f:
+        f.write(dedent("""
+        java_library(name='lib')
+        """))
+      pants_run = self.run_pants(['updates.ivy', '{}:lib'.format(tmpdir)])
+      self.assert_success(pants_run)
+
+      self.assertIn('Dependency updates available:', pants_run.stdout_data)
+      self.assertIn('All dependencies are up to date', pants_run.stdout_data)
```

Comment:
Does this mean that a target without jar dependencies will see both messages? I'm not sure I understand why that would be.

Also, I think it would be worthwhile to have a test asserting that the task isn't run if the targets do not include jar_libraries.